olxcleaner/loader/xml.py

```diff
@@ -109,12 +109,18 @@ def read_course(edxobj, node, directory, filename, errorstore, htmlfiles, pointe
                     empty = False
 
     # Check for a pointer tag
+    node_filename = os.path.splitext(os.path.split(filename)[-1])[0]
     if empty and edxobj.is_pointer(node.attrib):
-
         if pointer:
-            # The target of a pointer tag cannot be a pointer itself
-            errorstore.add_error(SelfPointer(filename,
-                                             edxobj=edxobj))
+            node_pointing_to_itself = node.attrib.get('url_name') == node_filename
+            if node_pointing_to_itself:
+                # The target of a pointer tag cannot be a pointer itself
+                pointer_error = SelfPointer
+            else:
+                # The target of a pointer has been pointed to and is unexpectedly empty.
```

Comment:
If we can leave the `else` part here, will it catch `InvalidPointerError` instead?